### PR TITLE
Set error_log location for containerization

### LINKED ISSUE #662: PHP error_log configuration is missing
`error_log` configuration is not set in PHP images. It should be set to  `/proc/self/fd/2` like [in the official PHP image](https://github.com/docker-library/php/blob/55fb15102c30235106e36e07f5eb447d54d16d27/8.1/buster/fpm/Dockerfile#L244-L257).
This is basically the default configuration for 12fa PHP application.

### PR DESCRIPTION
**Pull request by mxr576**
Closes #662

### CODE REVIEW

File: images/php-fpm/00-lagoon-php.ini.tpl [hunk 1]
```diff
@@ -8,6 +8,7 @@ display_startup_errors = ${PHP_DISPLAY_STARTUP_ERRORS:-Off}
 auto_prepend_file = ${PHP_AUTO_PREPEND_FILE:-none}
 auto_append_file = ${PHP_AUTO_APPEND_FILE:-none}
 error_reporting = ${PHP_ERROR_REPORTING:-E_ALL & ~E_DEPRECATED & ~E_STRICT}
+error_log=${PHP_ERROR_LOG:/proc/self/fd/2}
```
Comment (rocketeerbkw) [inline images/php-fpm/00-lagoon-php.ini.tpl:11]: To use a default value in [parameter expansion](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html), the syntax is `${parameter:-word}`, you're just missing the `-`.

**Review by rocketeerbkw — CHANGES_REQUESTED** [review]

File: images/php-fpm/00-lagoon-php.ini.tpl (hunk 1 above)
Comment (mxr576) [reply images/php-fpm/00-lagoon-php.ini.tpl:11]: Right, tx, I thought I have already forcr pushed a fix, but it looks like I did not. Coming tomorrow...

File: images/php-fpm/00-lagoon-php.ini.tpl [hunk 2]
```diff
@@ -8,6 +8,7 @@ display_startup_errors = ${PHP_DISPLAY_STARTUP_ERRORS:-Off}
 auto_prepend_file = ${PHP_AUTO_PREPEND_FILE:-none}
 auto_append_file = ${PHP_AUTO_APPEND_FILE:-none}
 error_reporting = ${PHP_ERROR_REPORTING:-E_ALL & ~E_DEPRECATED & ~E_STRICT}
+error_log=${PHP_ERROR_LOG:-/proc/self/fd/2}
```
Comment (mxr576) [inline images/php-fpm/00-lagoon-php.ini.tpl:11]: Hm, I have deployed the test image that was built by the CI from this PR to a Lagoon environment and in the CLI container I get

```
[foobar]test@cli-drupal:/app$ php -i | grep error_log
error_log => /proc/self/fd/2 => /proc/self/fd/2
```

but `admin/reports/status/php` says error_log is unset...

File: images/php-fpm/php-fpm.d/global.conf [hunk 3]
```diff
@@ -5,3 +5,5 @@
 ; Default Unit: seconds
 ; Default Value: 0
 process_control_timeout = 30s
+
+error_log = ${PHP_ERROR_LOG:-/proc/self/fd/2}
```
Comment (mxr576) [inline images/php-fpm/php-fpm.d/global.conf:9]: Let's try this...

Comment (mxr576) [reply images/php-fpm/php-fpm.d/global.conf:9]: Still the same...

**Comment by rocketeerbkw** [comment]
We think this is working correctly already.

**Pull request closed without merging**